src/interfaces/operation.ts

```diff
@@ -268,3 +269,26 @@ export interface OperationErrorEntity {
     fields: Record<string, string>,
     id: string
 }
+
+/**
+ * Action types after user confirm
+ */
+export const enum OperationPostInvokeConfirmType {
+    /**
+     * simple user confirm
+     */
+    confirm = 'confirm',
+}
+
+/**
+ * Action, which will be perform after user confirm
+ *
+ * @param type confirm type
+ * @param confirmText text for comfirm modal
+ * @param confirmTitle title for confirm modal
+ */
+export interface OperationPostInvokeConfirm {
```

Comment:
As we've discussed before, those interfaces are literally just a duplicate for existing preinvoke interfaces
https://github.com/tesler-platform/tesler-ui/blob/1b6279833905686c3c0e580b8c6414374336ed0e/src/interfaces/operation.ts#L83-L86

https://github.com/tesler-platform/tesler-ui/blob/1b6279833905686c3c0e580b8c6414374336ed0e/src/interfaces/operation.ts#L91-L107


They should be expanded if necessary and reused, because as we determine earlier preInvokes and postInvoke confirms are the same thing essentially, they just trigger differently (former by a field in operation rowMeta, latter by operation response postInvoke).